pinot-core/src/test/java/org/apache/pinot/core/startree/v2/PreAggregatedPercentileTDigestStarTreeV2Test.java

```diff
@@ -30,7 +30,9 @@
 
 public class PreAggregatedPercentileTDigestStarTreeV2Test extends BaseStarTreeV2Test<Object, TDigest> {
   // Use non-default compression
-  private static final double COMPRESSION = 50;
+  private static final double COMPRESSION = 200;
```

Comment:
Does this mean we need to change the compression factor to 200 to be within 5% error? IIRC, we are using 100?